### DISCUSSION TITLE
Requesting access to Broad instance of Seqr

### DISCUSSION THREAD

**Discussion by tommyli** [broadinstitute/seqr#1949] [Q&A]
How do I get access to Broad's instance of Seqr?  Ideally, this would only be for a test project but there is still sufficient data to play around.  It's convenient to see what's in the main/master branch ahead of time.

**Comment by hanars** [comment]
You need to register you account in [https://anvil.terra.bio/](https://anvil.terra.bio/) and then you will be able to sign in to the Broad's seqr instance using the "[sign in with google](https://seqr.broadinstitute.org/login)" option. We have an open ticket to make sample projects available to anyone who accesses seqr, so if you can wait a couple months you'll gain access to those.  If you need it earlier I can grant your account access to a couple of our test projects in the meantime

**Comment by tommyli** [comment]
I've registered with Anvil and managed to login with Google account.  Regarding the open ticket on demo projects, I presume you meant #1777 ?  Closing as I have no rush in getting access and will wait for #1777.  Thanks.

**Comment by hanars** [comment]
yes, that is the ticket I was referencing
